utils/firestore.js

```diff
@@ -431,25 +432,52 @@ const removeParticipantsDataDestruction = async () => {
                 ? new Date().getTime() - new Date(participant['269050420']).getTime()
                 : 0
 
-            if (participant['883668444'] === 704529432 || (Math.floor(timeDiiff / millisecondsInDay) > 60)) {
+            if (participant['883668444'] === 704529432 || (Math.floor(timeDiiff / millisecondsInDay) > 2)) {
```

Comment:
```suggestion
            if (participant['883668444'] === 704529432 || (Math.floor(timeDiff / millisecondsInDay) > 2)) {
```

A minor change, I think also the variable name should be changed to [`timeDiff`](https://github.com/episphere/connectFaas/pull/362/files#diff-be91e66430aa4dd2f61668d1bd17d3cb449f57947e13bfa2f6bc087919967282R431).

Reply:
Solved